acl.py

```diff
@@ -76,11 +76,23 @@ def build_groups(groups, team_id, **kwargs):
             group_id = json.loads(response.content)['groop']['id']
         group_members = json.loads(api.list_group_members(team_id, group_id, **kwargs).content)['members']
         for user in users['users']:
-            user_id = get_user_id(user['email'], team_members)
-            if user_id is None:
-                api.add_user_to_team(team_id, user['email'], **kwargs)
-            if user_id is not None and not user_in_group(user_id, group_members):
-                api.add_user_to_group(team_id, group_id, user_id, **kwargs)
+            if api.account_exists(user['email'], **kwargs):
+                user_id = get_user_id(user['email'], team_members)
+                if user_id is None:
+                    print 'Adding {} to team'.format(user['email'])
+                    api.add_user_to_team(team_id, user['email'], **kwargs)
```

Comment:
`get_user_id` looks at the list provided and returns the ID for the matching email address.  This informs as to whether the user is part of the team.  So if the user is on the team, `get_user_id` _should_ return the ID, otherwise `None`.